test-cases/features/scale-cluster.yaml

```diff
@@ -0,0 +1,22 @@
+test_duration: 30240
+
+cassandra_stress_threads: 500
+cassandra_stress_population_size: 10000000
+space_node_threshold: 100
+
+n_db_nodes: 180
+cluster_target_size: 200
```

Comment:
Please take these 2 lines out of this PR (possible also the duration if it's calculated based on this) put them in a configuration folder, with a yaml for the configuration e.g. scale-180-200.yaml and scale-450-500.yaml.

You can see a reference in KMIP PR from amos.
Then, you will need to add the new yaml to the test_config param in the pipeline file and have another file for the new test.

Reply:
@fruch does it seem like a correct usage?

Reply:
that could be a vaild usage of this options, if we really have two configuration we want to test regularly

Reply:
Done. Added 2 configuration yaml files and new pipeline